jet_tracking/gui/widgets/jetImageWidget.py

```diff
@@ -1,31 +1,116 @@
 import logging
+from PyQt5.QtWidgets import QGraphicsPixmapItem, QGraphicsView
+from tools.ROI import HLineItem, VLineItem, GraphicsScene
+from PyQt5.QtCore import Qt
+from PyQt5.QtGui import QPixmap, QImage
+import numpy as np
+from qimage2ndarray import array2qimage
```

Comment:
Is there a reason why qimage2ndarray must be used and not just Qimage from PyQt5.QtGui, which you have already loaded?
Here is an example of how to use it. 
https://stackoverflow.com/questions/60200981/pyqt5-convert-2d-np-array-to-qimage

Reply:
Lol, I tried so many times to deal with these image conversion issues over the years and it's a real headache. I encourage you to try if you would like. Tyler Pennebaker even brought up the same thing and gave it a whirl a while back and ended up agreeing with me that this module is the easiest way to do it and know it's done properly.